earth_enterprise/src/fusion/autoingest/sysman/AssetHandleD.h

```diff
@@ -39,6 +40,7 @@ class DerivedAssetHandleD_ : public virtual BaseD_, public ROBase_
   typedef ROBase_ ROBase;
```

Comment:
Q: I know this is not your code, but what's the purpose of defining type names ending in `_`, especially, if we're aliasing them to types without the underscore afterwards?

Reply:
It is good practice to typedef template parameters.  The standard library pretty much follows this practice too.  There is currently no way in C++ to extract this information external to the template class (when they add reflection then that might change).  Many times when using a template class externally this comes in handy

Reply:
Ah, yes, the C++ compiler needs a lot of explicit help.

Reply:
Well, actually there is a preference to using "using" in place of typedef in c++11 and later...